dashboard/config/scripts/levels/Music Level 1.level

```diff
@@ -9,6 +9,7 @@
       "toolbox": {
         "Play": ["play_sound_at_current_location_simple2"]
       },
+      "simpleToolbox": true,
```

Comment:
Minor: should we nest this inside the "toolbox" field? Something like
```
"toolbox": {
   "blocks": { "Play": [...] },
   "simple": true,
   // alternatively
   "type": "flyout" / "category"
}
```

Reply:
Good idea.  I was opting for legacy compatibility, but we're still at the stage where we should build the ideal format, and this is closer to that!